topics/engineering/running-background-processes-in-python.gmi

```diff
@@ -0,0 +1,68 @@
+# Running Background Processes in Python using Popen
+
+* author: bonfacem
+* reviewed-by: jnduli
+
```

Comment:
Can we restructure this document? I'd rather we provide the solution first and historical context later, because future consumers will most likely want to solve their problem first.